test/graphinfo.jl

```diff
@@ -57,6 +60,13 @@ single_parent_m = Model(μ = (1.0, () -> 3, :Logical), y = (1.0, (μ) -> MvNorma
 @test typeof(single_parent_m) == Model{(:μ, :y)}
 @test typeof(single_parent_m.g) == GraphInfo{(:μ, :y)}
 
+# test setindex
+
+@test_throws AssertionError setvalue!(m, @varname(s2), 0.0)
+@test_throws AssertionError setvalue!(m, @varname(s2), (1.0))
+setvalue!(m, @varname(s2), [1.0, 1.0])
+@test m[@varname s2].value[] == [1.0,1.0]
```

Comment:
Let's have paired getters and setters, and completly avoid exposing the internals:

```suggestion
@test getvalue(m, @varname(s2)) == [1.0,1.0]
```